tensorboard/uploader/uploader.py

```diff
@@ -425,12 +429,11 @@ def _run_values(self, run_to_events):
         for (run_name, events) in six.iteritems(run_to_events):
             for event in events:
                 v2_event = data_compat.migrate_event(event)
-                dataclass_events = dataclass_compat.migrate_event(
-                    v2_event, experimental_filter_graph=True
-                )
-                for dataclass_event in dataclass_events:
-                    if dataclass_event.summary:
-                        for value in dataclass_event.summary.value:
+                events = dataclass_compat.migrate_event(v2_event)
```

Comment:
Please don't shadow `events` (and `event` below)-- that is confusing and error-prone.  `dataclass_events` was fine imho.

Reply:
Just for what it's worth, all of this goes away in the subsequent PR anyway (https://github.com/tensorflow/tensorboard/pull/3511/files#diff-64ad30888691c0bf32cae63247f4ca5c).

Reply:
I remain opposed to any shadowing whatsoever, but don't need to argue this case since it's going away in #3511

(Eep, the former bug does suck, though; thanks for the fix).